internal/storage/postsql/read.go

```diff
@@ -705,7 +705,7 @@ func (r readSession) GetActiveInstanceStats() ([]dbmodel.InstanceByGlobalAccount
 		States: []dbmodel.InstanceState{dbmodel.InstanceNotDeprovisioned},
 	}
 
-	slog.Info("Calling - GetActiveInstanceStats", "filter", filter)
+	slog.Info("[last_operation_id] Calling GetActiveInstanceStats()")
```

Comment:
Can you use logger flags/fields?

Reply:
These logs will be removed tomorrow, the day after tomorrow probably.